docs/data/frontier-squid.md

```diff
@@ -143,6 +142,22 @@ If your squid will be supporting the Frontier application, it is also
 good to do the test in the
 [upstream documentation Testing the installation section](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Testing_the_installation).
 
+## Registering Frontier Squid
+
+See the [registration doc](/common/registration#new-registrations) for instructions on registering a new resource.
+(Or see the instructions for [updating existing registrations](/common/registration#updating-existing-registrations)
+if you are adding the squid service to an existing resource or resource group.)
+You will be using `template-resourcegroup.yaml` for your reference template to register various information about your
+resource.
+
+Additionally:
+
+- You will want to add a `Squid:` section to the `Services:` list, with any relevant fields for that service.
```

Comment:
Drop `you will want to`, it's in our list of superfluous statements https://opensciencegrid.org/technology/documentation/writing-documentation/#tips-for-writing-procedural-instructions